app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,30 @@
+function sendVote(qid, votestring) {
+  $.ajax({
+    url: '/questions/' + qid + votestring,
+    type: 'POST',
+    dataType: "JSON",
+    data: {"_method": "put"}
+  })
+    .done(function(data) {
+      $('#' + qid).text(data);
+    })
+    .fail(function() {
+      alert("Cannot vote for the same question more than once.");
+    });
+}
+
+function initVote() {
+  $('.upvote-button').unbind().click(function(ev) {
+    ev.preventDefault(); // prevents bootstrap from automatically adding a hidden attribute
```

Comment:
I actually think we don't need this `preventDefault()` line -- try removing this and verifying that everything works as expected. I think this is a holdover from some older code, and may not apply here. (Correct me if I'm wrong.)